backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -55,6 +56,28 @@ fun Route.configureShorkInterpreterControllerV1() {
         program.onSuccess { call.respond(Program(it)) }
         return@get
     }
+
+    post("/lobby") {
+        val lobbyId = shorkUseCase.createLobby()
+        val result = shorkUseCase.getLobbyStatus(lobbyId)
+
+        result.onFailure {
+            call.respond(HttpStatusCode.InternalServerError, it.message ?: UNKNOWN_ERROR_MESSAGE)
+        }
+        result.onSuccess { call.respond(HttpStatusCode.Created, lobbyId.toString()) }
+        return@post
+    }
+
+    get("lobby/status/{lobbyId}") {
+        val lobbyId =
+            call.parameters["lobbyId"]?.toLongOrNull()
+                ?: return@get call.respond(HttpStatusCode.BadRequest)
```

Comment:
I would like another empty line between the return and the rest of the code for readability purposes